rest_registration/locale/fr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,108 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-10-16 12:39-0400\n"
+"PO-Revision-Date: 2019-10-16 12:43-0400\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.0.6\n"
+
+#: rest_registration/api/serializers.py:162
+#: rest_registration/api/views/change_password.py:44
+msgid "Passwords don't match"
+msgstr "Les mots de passe ne correspondent pas"
+
+#: rest_registration/api/views/change_password.py:27
+msgid "Old password is not correct"
+msgstr "L'ancien mot de passe n'est pas valide"
+
+#: rest_registration/api/views/change_password.py:64
+msgid "Password changed successfully"
+msgstr "Changement du mot de passe réussie"
+
+#: rest_registration/api/views/login.py:43
+msgid "Login or password invalid."
+msgstr "Identifiant ou mot de passe non valide."
+
+#: rest_registration/api/views/login.py:47
+msgid "Login successful"
+msgstr "Connexion réussie"
+
+#: rest_registration/api/views/login.py:76
+msgid "Cannot remove non-existent token"
+msgstr "Ne peut supprimer un jeton non existant"
+
+#: rest_registration/api/views/login.py:78
+msgid "Logout successful"
+msgstr "Déconnexion réussie"
+
+#: rest_registration/api/views/register.py:89
+msgid "User without email cannot be verified"
+msgstr "Un usager sans mot de passe ne peut être vérifié"
+
+#: rest_registration/api/views/register.py:134
+msgid "User verified successfully"
+msgstr "Vérification de l'usager réussi"
+
+#: rest_registration/api/views/register_email.py:80
+#: rest_registration/api/views/register_email.py:134
+msgid "This email is already registered."
+msgstr "Ce courriel est déjà enregistré."
+
+#: rest_registration/api/views/register_email.py:94
+msgid "Register email link email sent"
+msgstr "Courriel du lien d'inscription envoyé"
+
+#: rest_registration/api/views/register_email.py:113
+msgid "Email verified successfully"
+msgstr "Vérification du courriel réussi"
+
+#: rest_registration/api/views/reset_password.py:110
+msgid "Reset password successful"
+msgstr "Réinitialisation du mot de passe réussi"
+
+#: rest_registration/exceptions.py:10
+msgid "Bad Request"
+msgstr "Mauvaise requête"
+
+#: rest_registration/exceptions.py:15
+msgid "User not found"
+msgstr "Usager non trouvé"
+
+#: rest_registration/notifications/email.py:161
+msgid "No 'subject' key found"
+msgstr "Aucune clé 'sujet' trouvé"
+
+#: rest_registration/utils/verification.py:18
+msgid "Signature expired"
+msgstr "Signature expirée"
+
+#: rest_registration/utils/verification.py:20
+msgid "Invalid signature"
+msgstr "Signature invalide"
+
+#~ msgid "email address"
```

Comment:
Any idea what's the commented definitions doing here? If they are not needed perhaps they should be removed?